docs/en/plugins/README.md

```diff
@@ -0,0 +1,413 @@
+
+First, create a new folder in the `/plugins` folder with the name of your plugin
+```
+mkdir plugins/loomio_tagging && cd plugins/loomio_tags
+```
+
+Then, update the `.gitignore` file to let it know you want to see your new plugin:
+
+```
+# .gitignore
+!plugins/loomio_tags
+```
+
+Now, make a `plugin.rb` file in the `plugins/loomio_tags` directory, and fill it with the following template:
+
+```
+# plugins/loomio_tags/plugin.rb
+module Plugins
+  module LoomioTags
+    class Plugin < Plugins::Base
+      setup! :loomio_tags do |plugin|
+        # our code will go here!
+        puts “Hello, world!”
+      end
+    end
+  end
+end
+```
+
+Now, if you run a command which boots the rails environment (like `rails c` or `rails s`), you’ll notice a friendly ‘Hello world!’ message. Congrats, you’ve made a plugin!
+
+Unfortunately, it’s not a terribly useful plugin yet, but we can make it better. First though, let’s push it up to a github repository. To do that, we’ll follow these steps:
+
+- Create a new git repository, like `git@github.com:loomio/loomio_tags.git`
+- Run `git init` in the `plugins/loomio_tags` folder
+- git remote add origin `git@github.com:loomio/loomio_tags.git`
+- `git add . && git commit -m “initial commit” && git push origin master`
+- (important, don’t skip!) add a reference to `plugins/plugins.yml`, telling Loomio where to find your plugin on github. That will look like this:
+
+```yaml
+  plugins:
+    loomio_tags:
+      repo: loomio/loomio_tags
+      version: master
+```
+
+(NB: you can constrain your plugin to a particular branch, tag, or commit with the version, but we’ll stick with tracking the master branch for now.)
+
+To prove that this works, we’ll nuke the local version of our plugin:
+
+```
+rm -rf plugins/loomio_tags
+```
+
+reacquire it:
+```
+rake plugins:acquire
+```
+
+and run a console again:
+```
+rails c
+```
+
+Which should result another friendly ‘Hello world’ message. Now we know Loomio can run our plugin from anywhere; woot!
+
+(NB: rake plugins:acquire can be a destructive action; it will delete all of the existing plugins on your local instance and replace them with those specified in `plugins.yml`. TL;DR you can easily lose your work if you skip step 5 above, don’t do it!)
+
+### PART 2: Add some simple functionality
+
+Alright, now we’ve got a working plugin, up on github. Time to make it do something a bit more useful! We’ll do this by issuing commands in our `setup!` method, which will either execute code on our instance directly, or reference other files which should be included in the app.
+
+##### 1. Tell Loomio to enable our plugin (important, don’t skip!)
+
+We’ll need to tell Loomio when to run this plugin code, otherwise it’ll skip everything and the plugin will be ‘off’ all the time. To do this, we can either provide an ENV variable to look at (the plugin will be enabled if the ENV is present)
+
```

Comment:
We’ll need to tell Loomio when to run this plugin code, otherwise it’ll skip everything and the plugin will be ‘off’ all the time. To do this, we can either provide an ENV variable to look at (the plugin will be enabled if the ENV is present)**:** [add colon]